common/flatpak-dir.c

```diff
@@ -7602,23 +7635,33 @@ flatpak_dir_install (FlatpakDir          *self,
           if (child_repo == NULL)
             return FALSE;
 
-          flatpak_flags |= FLATPAK_PULL_FLAGS_SIDELOAD_EXTRA_DATA;
+          bindfs_repo_file = g_file_new_for_path (bindfs_repo_path);
```

Comment:
this path may be NULL if the call above failed

Reply:
These bit is addressed in b55725c